openeo/internal/processes/parse.py

```diff
@@ -127,6 +127,12 @@ def from_json_file(cls, path: Union[str, Path]) -> Process:
         with Path(path).open("r") as f:
             return cls.from_json(f.read())
 
+    def get_parameter(self, parameter_id: str) -> Union[dict, list]:
```

Comment:
can you add a test for this new method?

Reply:
Also to stay closer to the openEO spec, I would just use `name` as argument here. "Parameter Id" is not really a thing in the spec